packages/render-fidelity-tools/package.json

```diff
@@ -19,8 +19,8 @@
     "node": ">=12.0.0"
   },
   "dependencies": {
-    "@babylonjs/loaders": "^4.1.0",
-    "@babylonjs/core": "^4.1.0",
+    "@babylonjs/loaders": "^4.2.0-alpha.22",
+    "@babylonjs/core": "^4.2.0-alpha.22",
```

Comment:
change to latest version

Reply:
I take it 4.2 is pre-release?

Reply:
Yeah, i will update it to form release when they have it